etc/document-definitions-loader.js

```diff
@@ -1,32 +0,0 @@
-/**
```

Comment:
This file has been moved to the `src` directory (see below). It is largely unchanged except that the `parseDocDefinitions` function was added to it.